planning/obstacle_cruise_planner/src/node.cpp

```diff
@@ -557,13 +557,15 @@ std::vector<Polygon2d> ObstacleCruisePlannerNode::createOneStepPolygons(
   const double step_length = p.decimate_trajectory_step_length;
   const double time_to_convergence = p.time_to_convergence;
 
-  std::vector<Polygon2d> polygons;
-  const double current_ego_lat_error =
-    motion_utils::calcLateralOffset(traj_points, current_ego_pose.position);
-  const double current_ego_yaw_error =
-    motion_utils::calcYawDeviation(traj_points, current_ego_pose);
+  const auto nearest_pose =
+    traj_points.at(*motion_utils::findNearestSegmentIndex(traj_points, current_ego_pose)).pose;
```

Comment:
@satoshi-ota Thank you for the comment.
Since there is no search constraints, I change the code to call the other `findNearestSegmentIndex()` which returns `size_t`